src/scss/typography.scss

```diff
@@ -1,3 +1,18 @@
+a {
+  @include theme-link-color;
+}
+
+.btn-primary {
+  @include theme-accent-color('background-color', 'darken', 10%);
+  @include theme-accent-color('border-color', 'darken', 10%);
+  transition: all 0.3s ease;
+}
+
+.btn-default {
+  @include theme-accent-color('border-color', 'darken', 10%);
+  transition: all 0.3s ease;
```

Comment:
This transition may be better placed in `.btn` since it's applied to `.btn-link, .btn-primary, .btn-default`